smoke-tests/grpc/build.gradle

```diff
@@ -17,8 +24,8 @@ dependencies {
   implementation "io.grpc:grpc-netty-shaded:1.32.1"
   implementation "io.grpc:grpc-protobuf:1.32.1"
   implementation "io.grpc:grpc-stub:1.32.1"
-  implementation "io.opentelemetry:opentelemetry-proto:0.9.1"
-  implementation "io.opentelemetry:opentelemetry-extension-auto-annotations:0.9.1"
+  implementation "io.opentelemetry:opentelemetry-proto:0.10.0-SNAPSHOT"
+  implementation "io.opentelemetry:opentelemetry-extension-auto-annotations:0.10.0-SNAPSHOT"
```

Comment:
is this change needed? smoke tests are passing on this PR, and I think(?) they use docker images from last build on master